streampipes-model-client/src/main/java/org/apache/streampipes/model/client/user/UserRegistrationData.java

```diff
@@ -18,32 +18,7 @@
 
 package org.apache.streampipes.model.client.user;
 
-public class Token {
-
-  private String principal;
-  private String credentials;
-
-  public Token(String principal, String credentials) {
-    super();
-    this.principal = principal;
-    this.credentials = credentials;
-  }
-
-  public String getPrincipal() {
-    return principal;
-  }
-
-  public void setPrincipal(String principal) {
-    this.principal = principal;
-  }
-
-  public String getCredentials() {
-    return credentials;
-  }
-
-  public void setCredentials(String credentials) {
-    this.credentials = credentials;
-  }
-
+import java.util.List;
 
+public record UserRegistrationData(String username, String password, List<String> roleNames) {
```

Comment:
Is it necessary to change the field to `roleNames`? Since the  `UserInfo` object also uses `roles`.